### ISSUE TITLE
Add auth:whoami --all endpoint

### ISSUE THREAD

**Issue by smt116** [deis/deis#4792]
Hi there,

TL;DR It would be great to add an endpoint for the `whoami` command (or add the new one) and make it return all user's informations from Deis.

Example Request:

```
GET /v1/whoami HTTP/1.1
Host: deis.example.com
Authorization: token abc123
```

Example Response:

```
HTTP/1.1 200 OK
DEIS_API_VERSION: 1.7
DEIS_PLATFORM_VERSION: 1.12.2
Content-Type: application/json

{
  "id": 1,
  "last_login": "2014-10-19T22:01:00.601Z",
  "is_superuser": true,
  "username": "test",
  "first_name": "test",
  "last_name": "testerson",
  "email": "test@example.com",
  "is_staff": true,
  "is_active": true,
  "date_joined": "2014-10-19T22:01:00.601Z",
  "groups": [],
  "user_permissions": []
}
```

## Background

We have a platform with multiple, independent Deis clusters. There is an application with API that synchronizes all informations and a plugin for Deis client. It allows developers to manage databases and their backups through that plugin and application.

## The problem

We would like to allow to destroy database only for application's owner and deny it to the contributors. We are using token to authenticate user against Deis cluster and check his permissions, but we are not able to recognize if he is an owner or not.

We don't want to use informations from the local profile file, because it may be manipulated by the user (you can change your local username by editing this file).

**Comment by bacongobbler** [comment]
this shouldn't be hard. How does `deis auth:whoami --all` sound to you?

**Comment by smt116** [comment]
Sounds good :)

**Comment by krancour** [comment]
I think this is a great idea, actually.  Given our current focus on v2 and @bacongobbler's estimation that this isn't too hard, I'm going to tag this as "help-wanted" and leave the implementation to the community.

The same feature should be ported forward to deis/workflow (the project containing the v2 CLI and controller).

**Comment by bacongobbler** [comment]
`deis users:list` displays all registered users as long as you're an admin. Closing.

**Comment by smt116** [comment]
@bacongobbler Have you read "The problem" section in the first post? I am not sure how showing a list of all users solves it. It is about translating user's token into name using Deis API as the name given in `~/.deis/foo.json` profile may be changed by user.

**Comment by bacongobbler** [comment]
Sorry! I misread it while going over as "make it return every user's information from Deis", not "make it return all of your information from Deis". My bad. Re-opening.

**Comment by smt116** [comment]
:+1:

**Comment by bacongobbler** [comment]
punting to v2 as this is a non-critical feature for v1's LTS release.
